From 910a589f904cfce8d6fe140c43134872d22a7eb0 Mon Sep 17 00:00:00 2001
From: Aparna M <a-m1@ti.com>
Date: Wed, 29 Sep 2021 16:22:34 +0530
Subject: [PATCH] setup-host-check: Modify script to support only Ubunut 18.04
 as host OS

Modify script to support only Ubunut 18.04 as host OS for am64xx-evm

Signed-off-by: Aparna M <a-m1@ti.com>
---
 setup-host-check.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup-host-check.sh b/setup-host-check.sh
index 45b4c71..df0d0b2 100644
--- a/setup-host-check.sh
+++ b/setup-host-check.sh
@@ -39,10 +39,10 @@ echo "Verifying Linux host distribution"
 
 get_host_type host
 
-if [ "$host" != "precise" -a "$host" != "trusty" -a "$host" != "xenial"  -a "$host" != "bionic" ]; then
-    echo "Unsupported host machine, only Ubuntu 12.04 LTS, Ubuntu 14.04 LTS, Ubuntu 16.04 LTS, and Ubuntu 18.04 LTS are supported"
+if [ "$host" != "bionic" ]; then
+    echo "Unsupported host Linux. Only Ubuntu 18.04 LTS is supported"
     exit 1
 fi
-echo "Ubuntu 12.04 LTS, Ubuntu 14.04, or Ubuntu 14.04 LTS is being used, continuing.."
+echo "Ubuntu 18.04 LTS is being used, continuing.."
 echo "--------------------------------------------------------------------------------"
 echo
-- 
2.17.1

